BaseFilter/sections/antiadblock.txt

```diff
@@ -791,6 +791,8 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/137611
+flasharch.com#@#.ad_area_wrap
```

Comment:
Where the rule came from? I can't even reproduce the issue.

Reply:
Okay, it's in French list. So the rule should be added to French. @smed79

Reply:
You have to move the rule to French list. It's not a Base issue.

Reply:
Moving ...

Reply:
Moved. https://github.com/easylist/listefr/pull/110